### PR TITLE
Feat - Add endpoint to receive server config

### PR DESCRIPTION
**Pull request by sebastien-lb**
linked to https://github.com/sebastien-lb/alfred-brain/pull/11

### CODE REVIEW

File: smart_lamp/lamp-test.py [hunk 1]
```diff
@@ -12,6 +12,24 @@
 def config():
     return getConfigResponse()
 
+@app.route('/serverConfig', methods=['POST'])
+def setServerConfig():
+    print(request)
+    print(request.get_json(force=True))
```
Comment (clement26695) [inline smart_lamp/lamp-test.py]: What does the method do if the format of the body is not JSON ?

File: smart_lamp/lamp-test.py [hunk 2]
```diff
@@ -12,6 +12,24 @@
...
+    data = request.get_json(force=True)
+    print(data)
```
Comment (clement26695) [inline smart_lamp/lamp-test.py]: Did you leave the print on purpose ?

Comment (sebastien-lb) [reply smart_lamp/lamp-test.py]: Not sure

File: smart_lamp/lamp-test.py (hunk 1 above)
Comment (sebastien-lb) [reply smart_lamp/lamp-test.py]: Bug probably

**Pull request merged**